src/NzbDrone.Core/Indexers/TitansOfTv/TitansOfTvParser.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using Newtonsoft.Json;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.Indexers.TitansOfTv
+{
+    public class TitansOfTvParser : IParseIndexerResponse
+    {
+        public IList<Parser.Model.ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
+        {
+            var results = new List<ReleaseInfo>();
+
+            if (indexerResponse.HttpResponse.StatusCode != HttpStatusCode.OK) return results;
+
+
+            var content = indexerResponse.HttpResponse.Content;
+
+            var parsed = JsonConvert.DeserializeObject<ApiResult>(content);

```

Comment:
please use, https://github.com/Terebi42/Sonarr/blob/develop/src/NzbDrone.Common/Serializer/Json.cs

we try to avoid using Json.NET directly.